packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -50,7 +50,7 @@ export const propTypes = {
   target: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
   isOpen: PropTypes.bool.isRequired,
   onClose: PropTypes.func.isRequired,
-  placement: PropTypes.oneOf(basePlacements),
+  placement: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
```

Comment:
Given the values are already hardcoded in the type and also the default value (bottom), I think it's ok to hardcode them here also.